utils/installer/install.sh

```diff
@@ -250,15 +250,28 @@ function setup_shim() {
   if [ ! -d "$INSTALL_PREFIX/bin" ]; then
     mkdir -p "$INSTALL_PREFIX/bin"
   fi
-  cat >"$INSTALL_PREFIX/bin/lvim" <<EOF
-#!/bin/sh
 
-export LUNARVIM_CONFIG_DIR="\${LUNARVIM_CONFIG_DIR:-$LUNARVIM_CONFIG_DIR}"
-export LUNARVIM_RUNTIME_DIR="\${LUNARVIM_RUNTIME_DIR:-$LUNARVIM_RUNTIME_DIR}"
+  local src="$LUNARVIM_RUNTIME_DIR/lvim/utils/bin/lvim"
+  local dst="$INSTALL_PREFIX/bin/lvim"
 
-exec nvim -u "\$LUNARVIM_RUNTIME_DIR/lvim/init.lua" "\$@"
-EOF
-  chmod +x "$INSTALL_PREFIX/bin/lvim"
+  # remove outdated installation so that `cp` doesn't complain
+  if [ -e "$dst" ]; then
+    if cmp --quiet "$src" "$dst"; then
+      rm "$dst"
+    fi
+  fi
+
+  cp "$src" "$dst"
+
+  # replace the default path with the new INSTALL_PREFIX
+  sed s"@LUNARVIM_RUNTIME_DIR=.\(.*\)@LUNARVIM_RUNTIME_DIR=\"$LUNARVIM_RUNTIME_DIR\"@"g \
```

Comment:
As suggested in the other PR, I would still prefer that it is known that these variables are replaced (also that the file is to be modified by another one).

`export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-{ LUNARVIM_RUNTIME_DIR }}"`

This would also allow to write a simpler sed command.

Reply:
That also means you can't copy paste the "default" binary, since it's a template now. I'm not sure if anyone expects it to be there. I don't really mind either way.